sw-precache-config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+  staticFileGlobs: [
+    'out/**/*.html',
```

Comment:
Finally, remove this because our `index.html` is not really a static file (it may be changed in the future). Otherwise, user may get stale pages.

We will let the runtime caching system do its work.